packages/cli/generators/extension/index.js

```diff
@@ -33,12 +33,12 @@ module.exports = class ExtensionGenerator extends ProjectGenerator {
 
   promptProjectName() {
     if (this.shouldExit()) return;
-    return super.promptProjectName();
+    return super.promptProjectName('Extension');
```

Comment:
Let's keep `project`, maybe `Extension project`.

Reply:
+1 to keep "project".

@hacksparrow if you find it confusing that both `lb4 app` and `lb4 extension` are using the same term, then perhaps use `Application project` and `Extension project`?

Reply:
@bajtos Yes, that would be better.